src/planscape/conditions/views.py

```diff
@@ -1,6 +1,9 @@
-from django.http import HttpRequest, HttpResponse, QueryDict
+from django.http import HttpRequest, HttpResponse, JsonResponse, QueryDict
 from django.db import connection
 
+import json, os
+
+PLANSCAPE_ROOT_DIRECTORY = '/Users/jessxlee/cnra/'
```

Comment:
This setting should be in the src/planscape/planscape/.env file.

Add it there, remove this line, add the imports

  from decouple import config
  from typing import cast

to the top, then set the directory to

  PLANSCAPE_ROOT_DIRECTORY = cast(str, config('PLANSCAPE_ROOT_DIRECTORY'))

Reply:
done!